security/guard_authentication.rst

```diff
@@ -412,11 +412,18 @@ Each authenticator needs the following methods:
     :class:`Symfony\\Component\\HttpFoundation\\Response` object that helps
     the user authenticate (e.g. a 401 response that says "token is missing!").
 
-**supportsRememberMe**
+**supportsRememberMe()**
     If you want to support "remember me" functionality, return true from this method.
     You will still need to active ``remember_me`` under your firewall for it to work.
     Since this is a stateless API, you do not want to support "remember me"
     functionality in this example.
+    
+**createAuthenticatedToken(UserInterface $user, string $providerKey)**
+    If you are implementing the :class:`Symfony\\Component\\Security\\Guard\\GuardAuthenticatorInterface`
+    class instead of extending the :class:`Symfony\\Component\\Security\\Guard\\AbstractGuardAuthenticator`
```

Comment:
Youre absolutely right. It seems that i am kind of tired after that week.... (in berlin since monday)